obidog/wrappers/doxygen_wrapper.py

```diff
@@ -4,9 +4,33 @@
 import tempfile
 
 from obidog.config import SOURCE_DIRECTORIES
+from obidog.logger import log
 
 
+DOXYGEN_PATH = os.environ.get("DOXYGEN_PATH", "doxygen")
+
+def _check_doxygen():
+    try:
+        with subprocess.Popen(
+            [DOXYGEN_PATH, "--version"], stdout=subprocess.PIPE
+        ) as doxygen_exec:
+            version = doxygen_exec.stdout.read().decode("utf-8").strip()
+            try:
+                version = version.split()[0].split(".")
+                if int(version[0]) >= 1 and int(version[1]) >= 8:
```

Comment:
`1.8.18` should be the minimum version due to certain [issues](https://github.com/doxygen/doxygen/issues/6319) with previous Doxygen versions